### PR TITLE
Feature/layout improvement

### PR DESCRIPTION
**Pull request by gabrielgalc**
#### What is the purpose of this pull request?
Improve layout of sidebar menu

#### What problem is this solving?
It makes the category menu cleaner.

#### How should this be manually tested?
https://galc--storecomponents.myvtex.com

#### Screenshots or example usage
screen shot 2018-11-28 at 6 04 33 pm

#### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/components/SideBarItem.js [hunk 1]
```diff
@@ -70,47 +71,49 @@ class SideBarItem extends Component {
...
+                  : <PlusIcon size={13} color={'currentcolor'} />
                 }
               </span>
             )
           }
         </div>
         {
           hasChildren && (
-            <div className="bg-light-silver">
+            <div>
```
Comment (luciannojunior) [inline react/components/SideBarItem.js]: This div becomes obsolet, I guess

File: react/components/SideBarItem.js [hunk 2]
```diff
@@ -70,47 +71,49 @@ class SideBarItem extends Component {
...
             {item.name}
           </span>
           {
             hasChildren && (
-              <span className={treeLevel === 1 ? 'c-muted-2' : 'c-muted-3'}>
+              <span className={treeLevel === 1 ? 'c-on-base' : 'c-muted-3'}>
                 {this.state.open
-                  ? <IconCaretUp size={13} color={'currentcolor'} />
-                  : <IconCaretDown size={13} color={'currentcolor'} />
+                  ? <DeleteIcon size={13} color={'currentcolor'} />
```
Comment (luciannojunior) [inline react/components/SideBarItem.js]: DeleteIcon? I would expect an trash can. `MinusIcon` would suit better, no?

File: react/images/DeleteIcon.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    size: 20,
+    fillColor: '#979899',
+  }
+
+  render() {
+    const { size, fillColor } = this.props
+    return (
+      <svg width={`${size}px`} height={`${size}px`} viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg">
+        <title>e delete</title>
+        <g fill={this.props.secondaryfill}>
```
Comment (luciannojunior) [inline react/images/DeleteIcon.js]: Put `secondaryfill` inside the destructuring statement on line 21, and also _camelCase_ it to `secondaryFill`

File: react/images/PlusIcon.js [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+    size: 20,
+    fillColor: '#3f3f40',
+  }
+
+  render() {
+    const { size, fillColor } = this.props
+    return (
+      <svg width={`${size}px`} height={`${size}px`} viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg">
+        <title>e add</title>
+        <g fill={this.props.secondaryfill}>
```
Comment (luciannojunior) [inline react/images/PlusIcon.js]: Same problems as above

File: react/components/SideBarItem.js [hunk 5]
```diff
@@ -1,10 +1,11 @@
 import React, { Component, Fragment } from 'react'
 import PropTypes from 'prop-types'
 import classNames from 'classnames'
-import { IconCaretDown, IconCaretUp } from 'vtex.styleguide'
 import { withRuntimeContext } from 'render'
 import { FormattedMessage } from 'react-intl'
 
+import PlusIcon from '../images/PlusIcon'
+import DeleteIcon from '../images/DeleteIcon';
```
Comment (luciannojunior) [inline react/components/SideBarItem.js]: Add a blank link after the last import

File: react/components/SideBarItem.js (hunk 1 above)
Comment (brunojdo) [reply react/components/SideBarItem.js]: @gabrielgalc Replace for this

```
hasChildren && this.state.open && ( 
<Fragment> 
...
```

**Comment by luciannojunior** [comment]
Telemarketing is getting above
image

**Pull request merged**